aws/route53/nameservers

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
```

Comment:
Is bash required here?

Reply:
Good spot! We need to remove bash indeed